### DISCUSSION TITLE
problem to simulate 100gr ethanol pool fire with water mist

### DISCUSSION THREAD

**Discussion by Hannanaaaaa** [firemodels/fds#12967] [Q&A]
Hallo everyone...
i have problem with my code, my temperature in this code [ETHANOL.txt](https://github.com/firemodels/fds/files/15390578/ETHANOL.txt) constant 27 celcius, can you help me to find my fault
thank you...

**Comment by drjfloyd** [comment]
Your input file does not run.  Fix the error and try running again.

 Starting FDS ...

 MPI Process      0 started on L-PW078R66

 Reading FDS input file ...


ERROR(605): SURF_ID POOL FIRE does not exist. (MPI Process: 0, CHID: METHANOL)

ERROR: FDS was improperly set-up - FDS stopped (CHID: METHANOL)

**Reply by Hannanaaaaa** [reply]
sorry i was wrong give a code, now i have edit it, thank you
